lib/taskana-core-test/src/test/java/acceptance/task/requestreview/RequestReviewAccTest.java

```diff
@@ -0,0 +1,131 @@
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+
+@TaskanaIntegrationTest
+public class RequestReviewAccTest {
+  @TaskanaInject TaskService taskService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup(ClassificationService classificationService, WorkbasketService workbasketService)
+      throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = DefaultTestEntities.defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_RequestReview_WhenTaskIsClaimed() throws Exception {
+    Task task = createTaskClaimedByUser_1_1().buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.requestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ForceRequestReview_WhenTaskIsClaimedByDifferentUser() throws Exception {
+    Task task = createDefaultTask().owner("user-1-2").buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.forceRequestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_RequestingReviewAndTaskIsClaimedByDifferentOwner()
+      throws Exception {
+    Task task =
+        createDefaultTask().owner("user-1-2").state(TaskState.CLAIMED).buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.requestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidOwnerException.class)
+        .hasMessageContaining("User 'user-1-1' is not owner");
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_TaskIsNotClaimedYet() throws Exception {
+    Task task = createDefaultTask().buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.requestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidTaskStateException.class)
+        .hasMessageContaining("[CLAIMED]");
```

Comment:
let's not verify the message. Please validate the instance variables of that exception.
Since InvalidTaskStateException contains multiple instance variables you have to extract it using
`catchThrowableOfType`. 
You can see an example @ CompleteTaskAccTest#should_ThrowException_When_UserIsNotAuthorizedOnTask

Reply:
Ok :)